entries/l/life360.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Life360": {
+    "domain": "life360.com",
+    "url": "https://www.life360.com/",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```